reference/release-notes.mdx

```diff
@@ -116,3 +116,34 @@ or (the non-shorthand version):
 ```
 [...] --secret key=value [...]
 ```
+
+# Breaking Change #2: Deprecation `path` and `port` for `liveness_probe`
+
+Starting from October of 2023, the usage of `path` and `port` for the `liveness_probe` will no longer be supported. Instead, the `command` option should be used for defining the liveness probe.
+
+## Who does this change affect?
+This change affects all users who are currently using the `path` and `port` options in their `liveness_probe` configuration. If you are one of them, please update your configuration to use the `command` option before October to avoid any disruptions in your application's health checks.
+
+## Why are we making this change?
+We are making this change to favor the `command` option, as it provides more flexibility in defining the liveness probe command.
```

Comment:
Mention some of the errors we've ran into regarding the liveness probe command here with path/port